pkg/frontend/openshiftclustercredentials_post_test.go

```diff
@@ -239,6 +258,10 @@ func TestPostOpenShiftClusterCredentials(t *testing.T) {
 			openshiftClusters := mock_database.NewMockOpenShiftClusters(controller)
 			subscriptions := mock_database.NewMockSubscriptions(controller)
 
+			subGetCalls := 1
```

Comment:
is subGetCalls actually needed - can you just wrap the subscriptions.EXPECT() with `if tt.doNotCheckSub` ?
almost tempted to reuse `tt.mocks != nil` and not even bother with doNotCheckSub.

Reply:
Right, `Times(0)` on an expected call is the same as not setting the expected call so we can just wrap it with `if`.

I was thinking about resusing `tt.mocks != nil`, but worried about potential cases where we need mocks, but don't need to check subscription. Now I tihnk - let's use `tt.mocks != nil` and modify it later, if such a case arise.